UDV-Core/src/Extensions/Contribute/src/CreateDocument.js

```diff
@@ -0,0 +1,243 @@
+export function CreateDocument(creationContainer, contributeController){
+
+  this.contributeController = contributeController;
+  this.creationContainer = creationContainer;
+
+this.myUploadedImage;
+  var docCreateButton = document.createElement('button');
+  docCreateButton.id = "docCreateButton";
+  var text = document.createTextNode("Create");
+  docCreateButton.appendChild(text);
+  document.getElementById("researchWindowTabs").appendChild(docCreateButton);
+
+  var docBrowserCreateButton = document.createElement('button');
+  docBrowserCreateButton.id = "docBrowserCreateButton";
+  var word = document.createTextNode("Create");
+  docBrowserCreateButton.appendChild(word);
+  document.getElementById("browserWindowTabs").appendChild(docBrowserCreateButton);
```

Comment:
Please avoid hardcoded dependencies when possible: if we change the id of the div containing the browser window, your code won't work anymore. Can you retrieve it from the document browser element ?

Reply:
ok